src/platforms/java/guides/spring-boot/index.mdx

```diff
@@ -141,7 +137,7 @@ public class CustomEventProcessor implements EventProcessor {
 
 ### Registering Custom Before Send Callback
 
-A Spring bean implementing `BeforeSendCallback` will be automatically set on `SentryOptions` during Sentry SDK auto-configuration. Note that there can be only single bean like that.
+A Spring bean implementing `BeforeSendCallback` will be automatically set on `SentryOptions` during Sentry SDK auto-configuration. Note that there can be only a single bean like that.
```

Comment:
A bit confusing, this might be more clear:
```suggestion
A Spring bean implementing the `BeforeSendCallback` will be automatically set on `SentryOptions` during the SDK's auto-configuration. Note that there can be only a single bean set this way.
```